test/hotspot/jtreg/vmTestbase/nsk/jdi/ListeningConnector/startListening/startlis001.java

```diff
@@ -103,16 +103,13 @@ private int runIt(String argv[], PrintStream out) {
          * matches the address which was set via connector's arguments.
          * Empty host address causes listening for local connections only (loopback interface).
          * */
-        String hostname = "localhost";
         List<String> validAddresses = new LinkedList<>();
-        validAddresses.add(hostname);
+        makeValidAddresses("localhost", validAddresses);
         try {
```

Comment:
Why try-catch is needed here?
The `UnknownHostException` has been already caught in `makeValidAddresses`.